src/test/worker-test-util.ts

```diff
@@ -15,7 +15,7 @@ import type {
 } from '../shared/worker-api.js';
 import type {CdnData} from './fake-cdn-plugin.js';
 
-const configureFakeCdn = async (
+export const configureFakeCdn = async (
```

Comment:
I read a comment somewhere mentioning how network errors are handled. Raised a small (non actionable) question.
Because we have a fake CDN we can possibly test or simulate various network errors. Do we test behavior under network failure. Very slow response or intermittent failure. I'm not sure how valuable that would be though.